src/main/com/intellij/lang/jsgraphql/ui/GraphQLSettingsConfigurable.kt

```diff
@@ -89,6 +89,13 @@ class GraphQLSettingsConfigurable(private val project: Project) :
                         settings::setFederationSupportEnabled
                     ).updateLibraries()
                 }
+                row {
+                    checkBox(
+                        "Apollo Kotlin",
```

Comment:
And here the message "graphql.library.apollokotlin" from the bundle is also missed 😉